app/backend/app.py

```diff
@@ -33,13 +32,13 @@
 from approaches.retrievethenread import RetrieveThenReadApproach
 from core.authentication import AuthenticationHelper
 
-CONFIG_OPENAI_TOKEN = "openai_token"
-CONFIG_CREDENTIAL = "azure_credential"
 CONFIG_ASK_APPROACH = "ask_approach"
 CONFIG_CHAT_APPROACH = "chat_approach"
 CONFIG_BLOB_CONTAINER_CLIENT = "blob_container_client"
 CONFIG_AUTH_CLIENT = "auth_client"
 CONFIG_SEARCH_CLIENT = "search_client"
+CONFIG_OPENAI_CHAT_CLIENT = "openai_chat_client"
+CONFIG_OPENAI_EMBEDDINGS_CLIENT = "openai_embeddings_client"
```

Comment:
Could you explain why you decided to keep two clients, even though they're the same object? I guess that's in case someone in the future wanted them to be different, it'd make it easier for them? It's not necessary now, right?

Reply:
Fixed